.github/workflows/diff.yaml

```diff
@@ -43,66 +43,79 @@ on:
         default: true
 
 jobs:
+  DiffSetup:
+    runs-on: ubuntu-latest
+    outputs:
+      run_community_core: ${{ steps.setup.outputs.run_community_core }}
+      run_coverage_core: ${{ steps.setup.outputs.run_coverage_core }}
+      run_debug_core: ${{ steps.setup.outputs.run_debug_core }}
+      run_debug_integration: ${{ steps.setup.outputs.run_debug_integration }}
+      run_jepsen_core: ${{ steps.setup.outputs.run_jepsen_core }}
+      run_release_core: ${{ steps.setup.outputs.run_release_core }}
+      run_release_benchmark: ${{ steps.setup.outputs.run_release_benchmark}}
+      run_release_e2e: ${{ steps.setup.outputs.run_release_e2e }}
+      run_release_stress: ${{ steps.setup.outputs.run_release_stress }}
+    env:
+      GITHUB_CONTEXT: ${{ toJson(github) }}
+      CONTEXT_FILE: github_context.json
```

Comment:
Huh?

Reply:
I will change `CONTEXT_FILE` to `CONTETS_FILE_NAME`